text/src/autogluon/text/text_prediction/predictor/predictor.py

```diff
@@ -292,60 +318,87 @@ def fit(self,
-        column_types, problem_type = infer_column_problem_types(train_data, tuning_data,
-                                                                label_columns=label_columns,
-                                                                problem_type=self._problem_type,
-                                                                provided_column_types=column_types)
-        self._eval_metric, log_metrics = infer_eval_log_metrics(problem_type=problem_type,
-                                                                eval_metric=self._eval_metric)
-        has_text_column = False
-        for k, v in column_types.items():
-            if v == _C.TEXT:
-                has_text_column = True
-                break
-        if not has_text_column:
-            raise AssertionError('No Text Column is found! This is currently not supported by '
-                                 'the TextPredictor. You may try to use '
-                                 'autogluon.tabular.TabularPredictor.\n'
-                                 'The inferred column properties of the training data is {}'
-                                 .format(column_types))
-        logger.info('Problem Type="{}"'.format(problem_type))
-        logger.info(printable_column_type_string(column_types))
-        self._problem_type = problem_type
-        if 'models' not in hyperparameters or 'MultimodalTextModel' not in hyperparameters['models']:
-            raise ValueError('The current TextPredictor only supports "MultimodalTextModel" '
-                             'and you must ensure that '
-                             'hyperparameters["models"]["MultimodalTextModel"] can be accessed.')
-        model_hparams = hyperparameters['models']['MultimodalTextModel']
-        self._backend = model_hparams['backend']
-        if plot_results is None:
-            plot_results = in_ipynb()
-        if self._backend == 'gluonnlp_v0':
-            import warnings
-            warnings.filterwarnings('ignore', module='mxnet')
-            from ..mx.models import MultiModalTextModel
-            self._model = MultiModalTextModel(column_types=column_types,
-                                              feature_columns=feature_columns,
-                                              label_columns=label_columns,
-                                              problem_type=self._problem_type,
-                                              eval_metric=self._eval_metric,
-                                              log_metrics=log_metrics,
-                                              output_directory=self._path)
+        if is_continue_training:
+            # We have entered the continue training / transfer learning setting. The model is not None and should
+            # have been loaded by a previous `TextPredictor.load()` call.
+            logger.info('Calling .fit() for a model that has already been loaded. Start the transfer learning setting.')
```

Comment:
"Start the transfer learning setting." is not totally clear what it means.
Do you mean "Training will continue starting from the existing model rather than from scratch."